sds_data_manager/utils/stackbuilder.py

```diff
@@ -145,21 +138,23 @@ def build_sds(
         #     ...
         # )
 
-        processing_stack.ProcessingStep(
+        # NOTE: processing_stack.ProcessingStep was giving this
+        # error:
+        # AttributeError: 'CfnJobDefinition' object has no attribute
+        # 'attr_job_definition_arn'
+        # Replaced it with batch job stack
```

Comment:
I'm not sure what this comment means in the context of the code. This seems more like a comment for the PR itself.

```suggestion
```